scripts/codecov-check.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+if command -v codecov > /dev/null 2>&1; then
+  echo The command codecov is available
+else
+  echo The command codecov is not available, installing...
+  set -x
+
+  echo Importing Codecov PGP public keys...
+  curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --import
+
+  echo Downloading codecov uploader...
+  curl -Os https://uploader.codecov.io/latest/linux/codecov
+
+  echo Downloading SHA signatures...
+  curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
+  curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
+
+  echo Verifying package integrity...
+  sha256sum -c codecov.SHA256SUM
+  gpg --verify codecov.SHA256SUM.sig codecov.SHA256SUM
```

Comment:
Don't we need a $? check here? Wouldn't this bash script just keep rolling even with an error?

Reply:
No, because of the `set -e` on line 2. That will cause it to exit immediately on any errors with a non-zero exit code.